### DISCUSSION TITLE
请问Milvus支持max，sum等聚合操作么？

### DISCUSSION THREAD

**Discussion by AliscaCL** [milvus-io/milvus#18105] [Q&A and General discussion]
请问Milvus支持max，sum等聚合操作么？
事情是这样的，因为需要根据Field中的最大值确定从哪里开始追数。
所以，就产生了，如何需要知道Milvus到底插到哪里了，需要计算sentence id的最大值的操作。
请问milvus支持这些操作么？

还有一点，milvus因为记录向量的成本非常高，所以我们把
vec+sentence_id 记录在milvus当中，然后还记录了一份，sentence_id + doc_id + insert_time 的关系。

请问milvus能够实现join一类的操作么？

因为先找到Topk的VEC，再通过时间窗口过滤的话，准确性会不如 先通过时间窗口过滤，再通过找Top K 的VEC。

**Comment by yhmo** [comment]
max/sum内置函数以及join操作，这些目前都还不支持。max/sum估计以后某个大版本会有（但近期的版本暂时不考虑）。join操作实现比较难，不确定以后会不会有。

PS: 鼓励用英文提问哈
